src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -145,13 +155,14 @@ private BoundBlock RewriteExpressionUsingStatement(BoundUsingStatement node, Bou
         }
 
         /// <summary>
-        /// Lower "using (ResourceType resource = expression) statement" to a try-finally block.
+        /// Lower "using (ResourceType resource = expression) statement" and
+        /// "using await (ResourceType resuorce = expression) statement" to a try-finally block.
```

Comment:
Typo: `resource`

Reply:
Perhaps short the comment to `Lower "using [await] (ResourceType resource = expression) statement" to a try-finally block.` Similar comment for previous method.